GTG/gtk/editor/editor.py

```diff
@@ -473,8 +473,17 @@ def refresh_editor(self, title=None, refreshtext=False):
         if self.task.parent:
             # Translators: Button label to open the parent task
             self.parent_button.set_label(_('Open Parent'))
-            # TODO: preview the parent task's title here:
-            self.parent_button.set_tooltip_markup(_('View the Parent Task'))
+            __tip_contents = _('View the Parent Task:')
+            __tip_contents += '\n<small><i>%s</i></small>' % self.task.parent.title
```

Comment:
I would prefer an [f-string ](https://docs.python.org/3/tutorial/inputoutput.html)here, since that's what we use in other places.

Reply:
Thank you for that suggestion! I have now done that in this rebased version.